src/app/listing/listing.page.html

```diff
@@ -52,6 +52,17 @@ <h2><b>{{solution?.name}}</b></h2>
                   <span class="certificate-tag">{{"CERTIFICATE"|translate}}</span>
                 }
               </div>
+              @if(listType == 'program'){
+                <div class="program-container">
+                    <h2 class="state">State : {{solution?.metaInformation?.state}}</h2>
```

Comment:
Why is State and Recommended for not translated?
Earlier it was there why was that removed?